src/xcode/ENA/ENA/Source/Extensions/Sequence+NSAttributedText.swift

```diff
@@ -0,0 +1,24 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+/** Helper to join array of NSAttributedString with a separator */
```

Comment:
Is, was this a multiline comment? Shouldn't it be changed to single line comment?